### ISSUE TITLE
Use character  "  to enclose cells containing column separators

### ISSUE THREAD

**Issue by drschoener** [opencultureconsulting/openrefine-client#28]
Sometimes quotation marks in data are a source of trouble. In the graphical user interface (create project, parse data as) I can control the behavior with the checkbox "Use character  "  to enclose cells containing column separators". It would be nice to have this feature also on the command line.

**Comment by felixlohmeier** [comment]
Yes, that's right. This option https://github.com/OpenRefine/OpenRefine/pull/1452 was introduced in OpenRefine 3.0 and is not yet supported by openrefine-client.

I am currently working on a new implementation of the client in Bash. https://github.com/opencultureconsulting/orcli already supports this option (but not much else).

```
orcli import csv "file" --quoteCharacter "+"
```
